<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QPANSOPYRNAVSIDMissedDockWidget</class>
 <widget class="QDockWidget" name="QPANSOPYRNAVSIDMissedDockWidget">
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>220</height>
   </size>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing"><number>1</number></property>
    <property name="margin"><number>2</number></property>

    <item>
     <widget class="QGroupBox" name="modeGroup">
      <property name="title"><string>Mode</string></property>
      <layout class="QHBoxLayout" name="modeLayout">
       <item>
        <widget class="QRadioButton" name="rnav1RadioButton">
         <property name="text"><string>RNAV1</string></property>
         <property name="checked"><bool>true</bool></property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rnav2RadioButton">
         <property name="text"><string>RNAV2</string></property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>

    <item>
     <widget class="QGroupBox" name="operationGroup">
      <property name="title"><string>Operation</string></property>
      <layout class="QHBoxLayout" name="opLayout">
       <item>
        <widget class="QRadioButton" name="sidRadioButton">
         <property name="text"><string>SID</string></property>
         <property name="checked"><bool>true</bool></property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="missedRadioButton">
         <property name="text"><string>Missed</string></property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>

    <item>
     <widget class="QGroupBox" name="inputGroup">
      <property name="title"><string>Input</string></property>
      <layout class="QFormLayout" name="inputLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="routingLabel">
         <property name="text"><string>Routing:</string></property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QgsMapLayerComboBox" name="routingLayerComboBox"/>
       </item>
      </layout>
     </widget>
    </item>

    <item>
     <widget class="QGroupBox" name="outputGroup">
      <property name="title"><string>Output</string></property>
      <layout class="QVBoxLayout" name="outputLayout">
       <item>
        <layout class="QHBoxLayout" name="outFolderLayout">
         <item><widget class="QLineEdit" name="outputFolderLineEdit"/></item>
         <item><widget class="QPushButton" name="browseButton"><property name="text"><string>...</string></property></widget></item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="exportKmlCheckBox">
         <property name="text"><string>KML</string></property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>

    <item>
     <widget class="QGroupBox" name="actionGroup">
      <property name="title"><string>Actions</string></property>
      <layout class="QVBoxLayout" name="actionLayout">
       <item>
        <widget class="QPushButton" name="calculateButton">
         <property name="text"><string>Calculate</string></property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>

    <item>
     <widget class="QGroupBox" name="logGroup">
      <property name="title"><string>Log</string></property>
      <layout class="QVBoxLayout" name="logLayout">
       <item>
        <widget class="QTextEdit" name="logTextEdit">
         <property name="readOnly"><bool>true</bool></property>
         <property name="minimumHeight"><number>100</number></property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>

   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 </ui>
